src/components/checkbox/index.tsx

```diff
@@ -272,10 +272,15 @@ class Checkbox extends Component<CheckboxProps, CheckboxState> {
 
   validate() {
     this.validationState = true;
+    // Validation: value must be true (checked)
     if (!this.props.disabled && !this.props.value) {
-      this.setState({isValid: false});
+      this.setState({valid: false});
     }
   }
+
+  isValid() {
+    return this.state.valid;
```

Comment:
Can't you keep the state variable as `isValid` (it is prettier), or do you not want to because you find it confusing?

Reply:
I thought it was confusing...